src/views/List/List.jsx

```diff
@@ -63,18 +86,65 @@ export default function List({ token }) {
     });
   };
 
+  //onChange handler for search input
+  const filterList = (e) => {
+    const { value } = e.target;
+    setSearchInput(value);
+    //the original data set is filtered so we always search from all the list items
+    let searchResults = data.filter((listItem) => {
```

Comment:
I see the filtering by search input happening twice on line 49 and 94 - to test I commented out the useEffect and the filter input still worked as expected -- maybe add this to refactoring review for the final sprint if you want more time to test the behavior

Reply:
Thank you Lindsey for pointing that out, it wasnt supposed to be a duplication.  The useEffect was an add on and I was supposed to move all the filtering functionality out of the filterList function into the useEffect so that the filtering didnt happen until after the state had time to update as the user typed into the search box.  What was happening was the filtering would be one character off, if you typed a whole word you didnt really notice, but if you only typed the first letter of a list item, the filter wouldnt start until you had typed a second letter even then it would only filter based off of just the first letter, not both, because that second letter hadnt been added to the state yet.  I removed the duplication and it still works the same!

Reply:
Perfect!